### PR TITLE
Add makim package

### PR DESCRIPTION
**Pull request by felipepaes**
This PR adds makim package recipe

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by felipepaes** [comment]
Hi @xmnlab, could you please confirm that it's okay to list you in the makim package recipe maintainers' list?

**Comment by xmnlab** [comment]
Thanks for adding me here as a maintainer @felipepaes !

**Comment by xmnlab** [comment]
@felipepaes for the test you will need to create a simple .makim.yaml file, otherwise it will fail.

I can give you more details soon

**Comment by xmnlab** [comment]
@felipepaes , please create a very simple .makim.yaml file in the recipes/makim directory and use this section to use that file for the tests: https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#source-files

I never used that .. but I guess it would fix this issue.
let me know if that works.

**Comment by felipepaes** [comment]
@xmnlab can I skip windows here as well?

**Comment by xmnlab** [comment]
@conda-forge/help-python , a friendly reminder about this PR, thank you :)

File: recipes/makim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/makim-{{ version }}.tar.gz
+  sha256: d7db0a978aa634d40896043057df11ea908d982080365e409b173c3e35116369
+
+build:
+  skip: True  # [win]
```
Comment (ocefpaf) [inline recipes/makim/meta.yaml:13]: This can probabley be noarch. `sh` should work on Windows bash, right?

Comment (xmnlab) [reply recipes/makim/meta.yaml:13]: mm not sure .. checking sh feedstock I saw this: https://github.com/conda-forge/sh-feedstock/blob/main/recipe/meta.yaml#L24

maybe we could remove this skip and add `__unix` similar to what `sh` is using.
maybe this is what you @ocefpaf meant?

@felipepaes could you try that please?

PS: this noarch and skip thing always get me confused XD

Comment (ocefpaf) [reply recipes/makim/meta.yaml:13]: Yep. You can make it noarch in the same fashion as `sh`. However, I'd love to test `sh` with a Windows shell, but that is an exercise for the future 😄

Comment (xmnlab) [reply recipes/makim/meta.yaml:13]: ohhh got it .. I didn't know that .. this is really new to me .. sorry for the confusion XD
we can try that as well .. if you have any example about that it we could try for sure

Comment (ocefpaf) [reply recipes/makim/meta.yaml:13]: You can use the `sh` package as an example.

Comment (xmnlab) [reply recipes/makim/meta.yaml:13]: awesome! ok thank you so much!

**Comment by felipepaes** [comment]
Thank you so much for your help @ocefpaf

**Comment by felipepaes** [comment]
Thank you so much for the review! @ocefpaf 🙏

**Pull request merged**